index.html

```diff
@@ -600,6 +600,13 @@ <h2>Protected Header Parameters</h2>
         <code>alg</code>, <code>iat</code>,
         <code>exp</code> and <code>cnf</code>.
       </p>
+      <p>
+        When <code>exp</code> or <code>exp</code> JWT claims are present,
+        they represent issuance and expiration time of a signature, respectively.
+        Note that this is different from <code>validFrom</code> and <code>validUntil</code>
```

Comment:
Note that they are different from the <code>validFrom</code> and <code>validUntil</code> properties

Reply:
```suggestion
        Note that they are different from <code>validFrom</code> and <code>validUntil</code> properties
```

Reply:
```suggestion
        Note that these are different from the <code>validFrom</code> and <code>validUntil</code> properties
```